src/sagemaker_sklearn_extension/feature_extraction/sequences.py

```diff
@@ -101,17 +108,25 @@ def __init__(
         self.augment = augment
         self.interpolation_method = interpolation_method
         self.extraction_type = extraction_type
+        self.extraction_seed = extraction_seed
+        self.sequences_lengths_q25 = sequences_lengths_q25
 
     def fit(self, X, y=None):
         X = check_array(X, dtype=None, force_all_finite="allow-nan")
+        if self.sequences_lengths_q25 is None or len(self.sequences_lengths_q25) != X.shape[1]:
+            raise ValueError(
+                f"length of sequences_lengths_q25 should be equal to number of columns in X (={X.shape[1]}).\n"
```

Comment:
Nit: Line break is not necessary

Reply:
I will fix it manually, but i think tox  added the line break here.

Reply:
sorry, ignore previous message, i realized you meant \n in the string -> I will delete it, good catch !